SwiftReflector/CustomSwiftCompiler.cs

```diff
@@ -214,7 +212,7 @@ void RemoveBadRPath (string library)
 			StringBuilder sb = new StringBuilder ();
 
 
-			sb.Append ("-xamreflect ");
+			sb.Append ("-xamreflect ").Append ("-enable-library-evolution ");
```

Comment:
Why not a single append? Just asking.

Reply:
In this case, adding things in argument by argument makes it less error prone if more args are added or removed.